array.c

```diff
@@ -3097,20 +3097,22 @@ rb_ary_to_s(VALUE ary)
  *  call-seq:
  *    to_a -> self or new_array
  *
- *  Returns +self+ if <tt>self.instance_of?(Array)</tt>:
+ *  When <tt>self.instance_of?(Array)</tt>, returns +self+:
```

Comment:
`When self is an instance of \Array, returns +self+:`

Reply:
Fixed.